user/serializers.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+
+
+class UserSerializer(serializers.ModelSerializer):
+    def validate(self, attrs):
+        data = super(UserSerializer, self).validate(attrs)
+        get_user_model().validate_password(
```

Comment:
The validate_password method is being used incorrectly. It doesn't take ValidationError as an argument. Instead, it raises a ValidationError if the password is invalid. You need to call this method in a try-except block and handle the ValidationError if it's raised.